File: aiko/src/main/java/com/aiko/domain/EquipmentPositionHistory.java
package com.aiko.domain;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.MapsId;
import javax.persistence.Table;
import javax.validation.constraints.NotNull;

import com.fasterxml.jackson.annotation.JsonIgnore;

import lombok.Data;

@Data
@Entity
@Table(name = "equipment_position_history", schema = "operation")
public class EquipmentPositionHistory implements Serializable{

	private static final long serialVersionUID = 1L;
	
	@JsonIgnore
	@Id
    private UUID id;

    @MapsId("id")
	@ManyToOne
	@JoinColumn(name = "equipment_id")
	private Equipment equipment;
	
	@NotNull
	private LocalDateTime date;
	@NotNull
	private BigDecimal lat;
	@NotNull
	private BigDecimal lon;

}

File: Api/Equipaments/Models/Equipment_Position_History.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Equipments.Models
{
    public class Equipment_Position_History
    {
        [Required]
        public Guid equipment_id { get; set; }
        [Required]
        public DateTime date { get; set; }
        [Required]
        public double lat{ get; set; }
        [Required]
        public double lon { get; set; }
    }
}
